src/components/Tooltip/Tooltip.tsx

```diff
@@ -386,13 +401,19 @@ const Tooltip = ({
   }, [])
 
   const hasContentOrChildren = Boolean(html || content || children)
+  const canShow = Boolean(
+    hasContentOrChildren &&
+      !calculatingPosition &&
+      (isOpen || show) &&
+      Object.keys(inlineStyles).length > 0,
```

Comment:
`Object.keys(inlineStyles).length > 0` this code with the small delay to show the tooltip (time to get the tooltip element rendered before position calculation) fixed the problem of tooltip start showing in a wrong place for the first time